### PR TITLE
Vpnaas: Endpoint group resource

### PR DESCRIPTION
**Pull request by simonre**
Part 4 out of 5 of a series that addresses #2 
@jtopjian I also put some minor doc fixes for the IPSec policy resource in here.

### CODE REVIEW

**Comment by simonre** [comment]
@jtopjian this is ready for review

File: openstack/resource_openstack_vpnaas_endpoint_group_v2.go [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+				ForceNew: true,
+				Computed: true,
+			},
+			"type": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+				Optional: true,
+			},
+			"endpoints": &schema.Schema{
+				Type:     schema.TypeList,
```
Comment (jtopjian) [inline openstack/resource_openstack_vpnaas_endpoint_group_v2.go]: Lists can only be used if the ordering is preserved. Neutron has a habit of returning lists in alphabetical/numerical order. Can you verify if this is the case here? In our acceptance tests, just reorder the endpoints and see if a diff is flagged.

If you specify the endpoints as `["10.3.0.0/24", "10.2.0.0/24"]` and neutron returns them differently, we'll need to change `TypeList` to `TypeSet`.

File: website/docs/r/vpnaas_endpoint_group_v2.html.markdown [hunk 2]
```diff
@@ -0,0 +1,69 @@
+---
+layout: "openstack"
+page_title: "OpenStack: openstack_vpnaas_endpoint_group_v2"
+sidebar_current: "docs-openstack-resource-vpnaas-endpoint-group-v2"
+description: |-
+  Manages a V2 Neutron Endpoint Group resource within OpenStack.
+---
+
+# openstack\_vpnaas\_endpoint_group_v2
```
Comment (jtopjian) [inline website/docs/r/vpnaas_endpoint_group_v2.html.markdown]: `openstack\_vpnaas\_endpoint\_group_v2`

I may have overlooked this in other resources :(

**Comment by simonre** [comment]
@jtopjian I rebased, fixed the issues and also set the "type" and "endpoints" attributes to ForceNew because they are not updateable.

**Comment by jtopjian** [comment]
@simonre Nice - thank you so much for verifying.

**Pull request merged**